Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FareParameters.hs

```diff
@@ -37,6 +37,9 @@ data FareParameters = FareParameters
     customerCancellationDues :: Maybe HighPrecMoney,
     tollCharges :: Maybe HighPrecMoney,
     congestionCharge :: Maybe HighPrecMoney,
+    insuranceCharge :: Maybe HighPrecMoney,
+    cardChargeOnFare :: Maybe HighPrecMoney,
```

Comment:
Why do we need two columns for cardCharge in fare params, at the end it will be one single value, right?